doc/nrf/protocols/bt/bt_mesh/dfu_over_bt_mesh.rst

```diff
@@ -182,8 +182,31 @@ To bring a stalled Target node to idle state, use the ``mesh models dfu cli canc
 
 .. _bluetooth_mesh_dfu_eval_md:
 
-Composing the firmware metadata
-*******************************
+Generating the firmware metadata
+********************************
+
+Automated metadata generation
+=============================
+
+The most efficient way to generate the required DFU metadata is to use the DFU metadata extraction script integrated in the NCS build system. By enabling the ``CONFIG_BT_MESH_DFU_METADATA_ON_BUILD`` option in the
+application, the metadata will be automatically parsed from the .elf and .config file. The parsed data is put into a DFU zip-archive in the build folder of the application in a JSON format. Additionally, the
+metadata string, required by the ``mesh models dfu metadata encode`` command, will be printed in the command line window when the application is built.
+
+.. note::
+   Currently this script only supports applications that contain a single composition data structure, meaning that applications containing multiple composition data structures must rely on manual generation of
+   the metadata string(s). Additionally, the script is hardcoded to produce a metadata string where the firmware is targeted for the application core.
+
+In addition to printing the metadata string on build, a separate west command can be utilized to print the metadata to the console, given that it is already generated by the build system. This provides the user
+easy access to this information without having to enter the .json file in the build folder or to rebuild the application::
+
+  west ble-mesh-dfu-metadata
+
+For this particular example, the output generated by either the build system or the west ble-mesh-dfu-metadata command would be::
+
+  Encoded metadata: 020000000100000094cf24017c26f3710100
```

Comment:
The complete script output should be added here:
https://github.com/nrfconnect/sdk-nrf/blob/7d209a57228062b2453d8ba25b172e8d09a26d49/scripts/bluetooth/mesh/mesh_dfu_metadata.py#L408-L410